samples/human_interaction/E4_SMSPhoneVerification/__init__.py

```diff
@@ -0,0 +1,38 @@
+import azure.durable_functions as df
+from datetime import timedelta
+
+
+def orchestrator_function(context: df.DurableOrchestrationContext):
+
+    phone_number = context.get_input()
+
+    if not phone_number:
+        raise Exception("A phone number input is required")
+
+    phone_number = int(phone_number)
```

Comment:
may want to try-catch around this to give a nice error message if someone provides a non-integer value. I imagine someone passing in a string with "-" in it is common.

Reply:
done!